code/svb-docker/python/svb_web/core/templates/internal/edit_customer.html

```diff
@@ -0,0 +1,31 @@
+{% extends "internal/base.html" %}
+
+{% block content %}
+<div class="container p-5">
+    <h1>Edit Customer</h1>
+    <div class="row">
+        <div class="col-sm">
+            <h1>{{form_title}}</h1>
+            <div class="mt-2 mb-2 {{existing_id_banner_display}}">
+                <button class="btn btn-primary mt-3 mb-3" id="print-button">Print ID</button>
+                <a href="/internal/accounts/edit" class="btn btn-secondary m-3">Create New Customer</a>
+            </div>
+            
+
+            <form enctype="multipart/form-data" action="" method="post">
+                {% csrf_token %}
+                <table style="table-layout: fixed; width: 100%; word-wrap: break-word">
+                    {{ form.as_table }}
+                </table>
+                <input class="btn btn-success mt-2" type="submit" value="Submit" />
+            </form>
+        </div>
+        <div class="col-sm p-5" style="min-width: 50%">
+            <img src="data:image/png;base64,{{id_card_image_data}}" class="img-fluid">

```

Comment:
NP:
```suggestion
            <img src="data:image/png;base64,{{ id_card_image_data }}" class="img-fluid">

```
also we don't have images now